value/less_test.go

```diff
@@ -25,301 +25,228 @@ func TestValueLess(t *testing.T) {
 		name string
 		// we expect a < b and !(b < a) unless eq is true, in which
 		// case we expect less to return false in both orders.
-		a, b Value
+		a, b interface{}
 		eq   bool
 	}{
 		{
-			name: "Invalid-1",
```

Comment:
Is it no longer possible to construct an invalid value? Is there a test that somehow verifies that of a Value's IsXXX() methods, exactly one always returns true?

Reply:
We could make a "IsInvalid" if that's useful. The content of Value can be litteraly anything, some things are invalid like ... `value.NewValueInterface(struct{})`.

Reply:
Which means I could re-add these tests

Reply:
Yeah, funcs, channels, structs, uints...

that last one should probably be fixed...

You might need a test object to make it easy to represent arbitrary invalid objects to put these tests back. The old Value actually might make that easy ;)

Reply:
Yeah, I'm fixing `uint`s right now, I'll add back the tests too